plugins/flat-object/src/test/java/org/opensearch/flatobject/xcontent/KeyValueJsonXContentParserTests.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.flatobject.xcontent;
+
+import org.junit.Test;
+import org.opensearch.common.bytes.BytesReference;
+import org.opensearch.common.xcontent.DeprecationHandler;
+import org.opensearch.common.xcontent.NamedXContentRegistry;
+import org.opensearch.common.xcontent.XContentBuilder;
+import org.opensearch.common.xcontent.XContentHelper;
+import org.opensearch.common.xcontent.XContentParser;
+import org.opensearch.common.xcontent.XContentType;
+import org.opensearch.common.xcontent.json.JsonXContent;
+import org.opensearch.flatobject.mapper.FlatObjectFieldMapper;
+
+import java.io.IOException;
+import java.util.logging.Logger;
+
+import static org.junit.Assert.assertEquals;
+
+public class KeyValueJsonXContentParserTests {
+    private static final Logger logger = Logger.getLogger((FlatObjectFieldMapper.class.getName()));
+
+    @Test
+    public void testKeyValueJsonXContentParserTests() throws IOException {
```

Comment:
It looks like this test is just testing the roundtrip logic through the existing XContent to/from JSON code path. It's not actually hitting your KeyValueJsonXContentParser code, right?

Reply:
yes, the tests are not added yet